### ISSUE TITLE
Publish gem on rubygems

### ISSUE THREAD

**Issue by dbourguignon** [newspaperclub/epdq#3]
Hi,

this gems will be handy for us.

Is there something preventing you of publishing it on rubygems ?

Thanks,
David

**Comment by adambeynon** [comment]
Agreed, any chance of a Gem release? Thanks for the work on it as well. Adam

**Comment by tomtaylor** [comment]
No, very happy to publish it, but I'm not actively using it at the moment, so I'd rather not be the maintainer. If someone else wants to take responsibility for it, I'm happy to allow access to this repo.
